ieeeuoft/makeuoft-2023
# fix: the text displayed on HSS is wrong when it is not signout period

Also, update the styling for the page showing the team code

## Steps to QA

- Change the hardware signout period to a future date, and visit the http://localhost:3000/acknowledgement page. Please make sure you made an application, accept yourself, and set your review a decision send date. 
- Change the hardware signout period to a past date, fill in the acknowledgment form, and see the new designed team code generation page.
